src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+document.querySelectorAll('.tree li').forEach((li) => {
+  const childNodes = Array.from(li.childNodes);
+
+  childNodes.forEach((node) => {
+    if (node.nodeType === Node.TEXT_NODE && node.textContent.trim() !== '') {
+      const span = document.createElement('span');
+
+      span.textContent = node.textContent.trim();
+      node.replaceWith(span);
+    }
+  });
+});
+
+document.querySelectorAll('.tree span').forEach((span) => {
```

Comment:
The event listener is correctly added to each span. Ensure that the spans are the only elements that should trigger this event to avoid unexpected behavior.